src/devices/bus/coco/coco_t4426.cpp

```diff
@@ -0,0 +1,195 @@
+// license:BSD-3-Clause
+// copyright-holders:Joakim Larsson Edstrom
+/***************************************************************************
+ *
+ *  coco_t4426.c
+ *
+ *  Terco T4426 CNC Programming Station multi cart
+ *
+ *  The code here is heavily inspired by coco_pak and coco_232
+ *
+ *  +-------------------------------------------------------------------------------+
+ *  ||__|+-----+    oo   75                               |O ||||||||||||||| O|     |
+ *  |XTL||MC   |    oo  110                                                         |
+ *  |1.8||14411|    oo  ..        +--+ +--+                                         |
+ *  |432||BRG  |    ==  600       |74| |74| +------+                                |
+ *  |MHz||     |    oo  ..        |LS| |LS| |MC1488|                                |
+ *  +---+|     |    oo 7200       |139 |00| +------+                                |
+ *  |    +-----+    oo 9600       +--+ +--+                    +--+                 |
+ *  |                                      +-------------+     |MC|                 |
+ *  |    +-----+   +-----+    +-----+      | EF68B50P    |     |14|                 |
+ *  |    | 2764|   | 2764|    |     |      | ACIA        |     |89|                 |
+ *  |    |     |   |     |    |     |      +-------------+     +--+                 |
+ *  |    |CA   |   |CA   |    |PMOS |                                               |
+ *  |    | 4426|   | 4426|    | 4426|   +-------------------+                       |
+ *  |    |  -6 |   |  -7 |    |     |   |  EP68B21P         |                       |
+ *  |    |     |   |     |    |     |   |  PIA              |                       |
+ *  |    |     |   |     |    +-----+   +-------------------+                       |
+ *  |    +-----+   +-----+                                                          |
+ *  |    +-----+   +-----+    +-----+   +-----+    +-----+   +-----+                |
+ *  |    | 2764|   | 2764|    | 2764|   | 2764|    | 2764|   | 2764|                |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |                |
+ *  |    |CA   |   |CA   |    |PD   |   |PD   |    |ED   |   |ED   |                |
+ *  |    | 4426|   | 4426|    | 4426|   | 4426|    | 4426|   | 4426|                |
+ *  |    |  -5 |   |  -4 |    |  -3 |   |  -2 |    |  -1 |   |  -0 |                |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |       OO       |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |                |
+ *  |    +-----+   +-----+    +-----+   +-----+    +-----+   +-----+                |
+ *  +-------------------------------------------------------------------------------+
+ *
+ ***************************************************************************/
+
+#define VERBOSE 0
+
+#define LOGPRINT(x) do { if (VERBOSE) logerror x; } while (0)
+#define LOG(x)      {} LOGPRINT(x)
```

Comment:
Yep, that's why I added the extra curly brackets which disabled the semicolon check as discussed previously. What I wanted to start with is to distinguish debug statements from plain code by putting them in a macro. In some places this hasn't been enough as I also needed some variables to keep values between calls to avoid flooding the debug console etc. 

I have now accepted that the compiler actually remove code in the body of

if (LOGME) {.....}

instead of 

LOGME(.....)

So I can clean up a lot if this and will take some time to do that at some point.